doc/man1/openssl-pkcs12.pod

```diff
@@ -76,22 +76,16 @@ by default.
 The filename to write certificates and private keys to, standard output by
 default.  They are all written in PEM format.
 
-=item B<-passin> I<arg>
+=item B<-passin>, I<arg>, B<-passout> I<arg>
 
-The PKCS#12 file (i.e. input file) password source. For more information about
-the format of B<arg>
+The password source for the input and output file.
+For more information about the format of B<arg>
```

Comment:
The old `-passout` text correctly specifies that the passphrase is applied on private keys, not the whole output file (which will consist of a pile of certificates and private keys, and the certificates will not be encrypted).  Please rephrase.

Reply:
ouch.  you're right.  i'll have to  think about how to do that in common text, if possible.

i think only key material is encrypted via ```-passout``` ?

Reply:
fixed.  and there were two copies of this item, i removed one.